### DISCUSSION TITLE
0.0.24 typo/bug

### DISCUSSION THREAD

**Discussion by slyoldfox** [astrandb/miele#136] [General]
@astrandb  I was reading through the changes from 0.0.23 to 0.0.24 and found this typo/bug

https://github.com/astrandb/miele/blob/27188e5a864d3d605f8a8db0a1afa26172d04e5d/custom_components/miele/binary_sensor.py#L317

**Comment by astrandb** [comment]
Thank you for reporting. I shall fix immediately.
